core/src/main/java/org/elasticsearch/Version.java

```diff
@@ -74,15 +74,17 @@
     public static final Version V_5_3_2 = new Version(V_5_3_2_ID, org.apache.lucene.util.Version.LUCENE_6_4_2);
     public static final int V_5_4_0_ID = 5040099;
     public static final Version V_5_4_0 = new Version(V_5_4_0_ID, org.apache.lucene.util.Version.LUCENE_6_5_0);
-    public static final int V_5_5_0_ID_UNRELEASED = 5050099;
```

Comment:
I almost certainly should have waited to make this change as a purely mechanical followup. It *nasty* to backport. But I already bit this pill so I feel like I have to swallow it now....